Bottom: 6db1abfbaf27fd517b2241c0a9ebe91b11094c22
Top:    43b1871181ef1ab99635be105043917df7b3874a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

No need to delete /etc/mailname before overwrite


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index a99acea..7080bd4 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -25,8 +25,6 @@ case ${1} in
           && [[ ! -f ${CONFIG_PATH}/exim4.conf ]]
         then
             # set the hostname
-            [ -e /etc/mailname ] \
-                && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
             # set local IP addr
             LOCAL_IP_ADDR=$(getent hosts $HOSTNAME | cut -d ' ' -f 1)
